src/Authentication/Authentication.Core/Microsoft.Graph.Authentication.Core.csproj

```
@@ -11,7 +11,7 @@
     <EnforceCodeStyleInBuild>true</EnforceCodeStyleInBuild>
   </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="Azure.Identity" Version="1.10.2" />
+    <PackageReference Include="Azure.Identity" Version="1.9.0" />
     <PackageReference Include="Azure.Identity.BrokeredAuthentication" Version="1.0.0-beta.3" />
```

`Azure.Identity.BrokeredAuthentication` was renamed to `Azure.Identity.Broker` - https://www.nuget.org/packages/Azure.Identity.BrokeredAuthentication:
```suggestion
    <PackageReference Include="Azure.Identity.Broker" Version="1.0.0-beta.5" />
```